app/Events/OfferPriceChanged.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Events;
+
+use App\Models\Offer;
+use Illuminate\Broadcasting\Channel;
+use Illuminate\Broadcasting\InteractsWithSockets;
+use Illuminate\Broadcasting\PresenceChannel;
+use Illuminate\Broadcasting\PrivateChannel;
+use Illuminate\Contracts\Broadcasting\ShouldBroadcast;
+use Illuminate\Foundation\Events\Dispatchable;
+use Illuminate\Queue\SerializesModels;
+
+class OfferPriceChanged
+{
+    use Dispatchable, InteractsWithSockets, SerializesModels;
+
+    /**
+     * Create a new event instance.
+     */
+    public Offer $offer;
+    public function __construct(Offer $offer)
+    {
+        $this->offer = $offer;
+    }
+
+    /**
+     * Get the channels the event should broadcast on.
+     *
+     * @return array<int, \Illuminate\Broadcasting\Channel>
+     */
+    public function broadcastOn(): array
```

Comment:
Jeśli nie zamierzasz korzystać z websocketów to raczej proponuję to usunąć, bo jeszcze kiedyś coś gdzieś poleci :)